package.json

```diff
@@ -41,7 +41,7 @@
   "peerDependencies": {
     "@apollo/client": "^3.3.19",
     "graphql": "^15.5.0",
-    "react": "^16.8.0 || ^17.0.0"
+    "react": "^18.2.0"
```

Comment:
**suggestion**: I think we should have here all ranges of react

The current dashboard still uses React 17.

Reply:
That's what I thought initially but this package is already deprecated and this last update will be used only for repos updated to React v18

Reply:
I'm wondering if in the future we don't want to have bugfix and if we migrate to React 18 we will have to upgrade dashboard as well - that is why I think it will be better if we support both 17 and 18 🤔

Reply:
version updated